guides/process/scheduling/how_to_timesheet.md

```diff
@@ -40,6 +40,12 @@ You may be asked to support internal Made Tech initiatives while on a billed eng
 
 Where these commitments total less than 5 hours per week, they should be included in our standard operating overhead and should not be accounted for separately from billed hours.
 
+Note that some internal activities do not count towards this 5 hours as our clients are perceived to benefit from them indirectly. These include:
+
+* Hiring
+* Line management (1:1s etc.)
+* Taking part in communities of practice
+
 When the commitment exceeds 5 hours in a week, each element should be allocated individually to the appropriate project in Kimble and not included in billable hours.
```

Comment:
It is not clear here whether we start allocating elements above 5 hours, or the whole lot.

If we do 7 hours, to we allocate 7 hours to non client categories or 2 hours?

Reply:
```suggestion
When the total internal commitment exceeds 5 hours in a week, each commitment should be allocated in its entirety to the appropriate project in Kimble and not included in billable hours.
```

Reply:
How about this?
```suggestion
When the commitment exceeds 5 hours in a week the 5 hour rule no longer applies, so each element should be allocated individually to the appropriate project in Kimble and not included in billable hours.
```

Reply:
Mashup of mine and @adrianclay's...
```suggestion
When the total internal commitment exceeds 5 hours in a week the 5 hour rule no longer applies, so each commitment should be allocated in its entirety to the appropriate project in Kimble and not included in billable hours.
```

Reply:
I'm going with my mashup.